### PR TITLE
Add licensing guidelines to ReadtheDocs

### PR DESCRIPTION
**Pull request by bourque**
This PR moves the licensing guideline to under the `sphinx` source tree, thus allowing it to render in ReadTheDocs. I also made a few style changes to the licensing markdown file, and enabled support for markdown in the sphinx config.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -14,15 +14,24 @@
...
+    'sphinx.ext.napoleon',
+    'sphinx.ext.viewcode',
+]
 
 templates_path = ['_templates']
 exclude_patterns = []
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.md': 'markdown',
+}
```
Comment (bourque) [inline docs/source/conf.py:29]: Was super happy to see how easy it was to enable both restructuredtext and markdown! Yay `sphinx`!

File: docs/source/conf.py [hunk 2]
```diff
@@ -14,15 +14,24 @@
...
 
 
 
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```
Comment (bourque) [inline docs/source/conf.py:36]: Going with this for now, but definitely open to ideas for other themes we might want to use.

File: docs/source/licensing.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+To avoid copyright concerns, it is recommended that:
+1. Any software created by LASP is properly licensed to provide sufficient guidance on their usage
+2. Any software used by LASP are licensed and used accordingly to protect against potential legal action from the owner
+   of that intellectual property
+
+The following table is taken from [The Turing Way](https://the-turing-way.netlify.app/reproducible-research/licensing)
+and shows the different categories of licenses available and the types of licenses that support their respective
+requirements.
+
+![The different categories of licenses](_static/licenses.png)
```
Comment (bourque) [inline docs/source/licensing.md:24]: I was having lots of trouble trying to get HTML to render in `sphinx`/ReadtheDocs, so I decided to just show a static image of the table instead 🤷🏻 . Maybe we can get embedded HTML working in a future PR?

File: docs/source/conf.py (hunk 1 above)
Comment (vmartinez-cu) [reply docs/source/conf.py:29]: Yay! This is awesome

**Review by vmartinez-cu — APPROVED** [review]
Looks great! Thanks for doing this

**Comment by vmartinez-cu** [comment]
Just noticed that the Read the Docs build failed. Looks like an issue with the module 'myst_parser'

**Comment by bourque** [comment]
I think I solved the RTD build issues. These updates are now 'live' here: https://lasp-developer-guide--21.org.readthedocs.build/en/21/licensing.html

File: docs/source/licensing.md (hunk 3 above)
Comment (vmartinez-cu) [reply docs/source/licensing.md:24]: Looks like the table caption is above the table again. We might want to re-make that png and add the caption in the .md file below the image at some point

**Review by kjbrooks — APPROVED** [review]

**Pull request merged**